docs/install.md

```diff
@@ -1,28 +1,47 @@
 #Installation and Settings
+##Prerequisites
 The MongoDB module and sub-modules need some configuration to be useful. This
 guide assumes that :
 
-* a [MongoDB][download] 3.0 to 3.6 instance is already installed, configured, and
+* a [MongoDB][download] 3.0 to 4.0 server instance is already installed, configured, and
   available to connect to from the Drupal instance.
 * the site will be running [Drupal][drupal] 8.[56].x, with [Drush][drush] 8.x.
 * the [mongodb][mongodb] (not [mongo][mongo]) PHP extension version 1.1.7 or
   later is installed and configured.
 * PHP is version 7.[01].x. At this point, [PHP 7.2.x][php72] might not pass the
   test suite. It should be compatible by the time Drupal 8.6.0 is released.
 
+**NOTE** : There is a plan to support Drush 9.x and it can be tracked [here][drush9]
+
+In a nutshell, any MongoDB server >=3.6, PHP extension >= 1.3.4, MongoDB PHP library >= 1.2.0 and PHP version >= 7.1. We recommend using composer for installing this module. For details refer [below](#composer-requirements).
+
+For more details about MongoDB extension and library for PHP check [here][PHPMongoDBext]
+
+ Some more links for you:
+
+   * [MongoDB Mac installation][MongoDBMac]
+   * [MongoDB LINUX installation][MongoDBLinux]
+   * [MongoDB Windows installation][MongoDBWindows]
+
 [download]: https://www.mongodb.org/downloads
 [drupal]: https://www.drupal.org/project/drupal
 [drush]: https://www.drupal.org/project/drush
 [php]: http://php.net/downloads.php
 [mongo]: http://php.net/mongo
 [mongodb]: http://php.net/mongodb
 [php72]: https://www.drupal.org/node/2936045
+[drush9]: https://www.drupal.org/project/mongodb/issues/2986785
+[PHPMongoDBext]: http://php.net/mongodb
+[MongoDBMac]: https://docs.mongodb.com/manual/tutorial/install-mongodb-on-os-x/
+[MongoDBLinux]: https://docs.mongodb.com/manual/administration/install-on-linux/
+[MongoDBWindows]: https://docs.mongodb.com/manual/tutorial/install-mongodb-on-windows/
 
-If MongoDB 3.0 to 3.5 is installed on `localhost:27017` and `mongod` was started
+If MongoDB 3.0 to 4.0 is installed on `localhost:27017` and `mongod` was started
```

Comment:
No: this featured was removed in to 3.6, so this is really 3.0 to 3.5 only. Maybe add a note explaining this is not a doc bug but it really stops at 3.5 ?